.github/workflows/comment.yaml

```diff
@@ -35,6 +35,7 @@ on:
       - "StepZen Test"
       - "Strawberry GraphQL Test"
       - "Swift Graphiti Test"
+      - "Ballerina GraphQL Test"
```

Comment:
please preserve alphabetic order so it is easier to maintain

Reply:
Fixed with [577a548](https://github.com/apollographql/apollo-federation-subgraph-compatibility/pull/474/commits/577a5487491c8e8644b7c1987ca85c367aa5ae08)